AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -93,6 +93,10 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+fiveguys.co.uk###js-cookie-overlay
+fiveguys.co.uk#$##cookieModal { display: none !important; }
+fiveguys.co.uk#$#.fancybox-container { display: none !important; }
+fiveguys.co.uk#$#body { overflow: visible !important; margin-right: 0px !important; }
```

Comment:
let's use `fiveguys.co.uk#%#//scriptlet("set-cookie", "5g_cookies", "1")` instead of ext-CSS

Reply:
@wout-codes